app/views/contents/_show.html.erb

```diff
@@ -1,36 +1,66 @@
 <%= turbo_frame_tag content, 'show' do %>
   <%= turbo_stream_from 'show', content %>
   <% content_files ||= content.content_files.path_order.page(1) %>
```

Comment:
i'm not sure why this is this here -- content_files should always exist coming from the controller?